recipes/mosh/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mosh" %}
+{% set version = "1.2.6" %}
+{% set sha256 = "7e82b7fbfcc698c70f5843bb960dadb8e7bd7ac1d4d2151c9d979372ea850e85" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://mosh.org/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+    - protobuf
+    - ncurses 5.9*
+    - zlib 1.2.*
+    - openssl 1.0.*
+  run:
+    - protobuf
+    - ncurses 5.9*
+    - zlib 1.2.*
+    - openssl 1.0.*
```

Comment:
To make sure `openssl` is picked add `--with-crypto-library=openssl` to `./configure` call